root/instrument/List.js

```diff
@@ -27,7 +27,12 @@ const Instrument = ({instrument}) => (
       ? (
         <>
           {' — '}
-          {l_instrument_descriptions(instrument.description)}
+          <span
+            dangerouslySetInnerHTML={{
+              __html:
+                l_instrument_descriptions(instrument.description),
```

Comment:
Sorry, I was dumb for suggesting `dangerouslySetInnerHTML`. We should actually be using:

```JSX
{expand2react(l_instrument_descriptions(instrument.description))}
```
(`expand2react` has to be imported, and a `<span>` wrapper is no longer needed.) This is safer since it only parses a safe subset of HTML.

Reply:
Done :)